examples/006_unbinned_fit/unbinned.py

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
 """
 kafe2 example: Unbinned Fit
 ================================
 
-An unbinned fit is needed, when there are too few data points to create a (good) histogram. If a probability density
-function (pdf) is fitted to a histogram generated from too few data points, the result won't be accurate. With an
-unbinned likelihood fit it's still possible to fit the pdf to the data points, as the likelihood and not the shape of
-the histogram is fitted.
+An unbinned fit is needed, when there are too few data points to create a (good) histogram. If a histogram is created
+from too few data points, information can be lost or even changed, by changing the exact value of one data point to the
+range of a bin. With an unbinned likelihood fit it's still possible to fit the pdf to the data points, as the likelihood
+of each data point is fitted.
 
 In this example the decay time of myons is calculated. Tha data was collected using the Kamiokanne-Experiment (a
```

Comment:
Kamiokanne-Experiment -> Kamiokanne experiment (I think)

Reply:
Should be "Kamiokanne experiment". Also, *muon* is spelled with a "u".